timeseries/src/autogluon/timeseries/metrics/abstract.py

```diff
@@ -242,12 +241,12 @@ def check_get_horizon_weight(horizon_weight: None, prediction_length: int) -> No
     @staticmethod
     def check_get_horizon_weight(
         horizon_weight: Union[Sequence[float], np.ndarray], prediction_length: int
-    ) -> npt.NDArray[np.float64]: ...
```

Comment:
this caused issues with a newer install, so removing for consistency also with the rest of the library